docs/quick-reference.md

```diff
@@ -167,6 +167,22 @@ network's most recent checkpoint, but this behaviour can be further modified usi
 cause the node to start with a fast in-memory catchup to ledger `N` with hash `HEXHASH`, and then
 replay ledgers forward to the current state of the network.
 
+A stateless and meta-streaming node can additionally be configured with
+`UNSAFE_EMIT_META=false` (if unspecified, the default is `true`).  If
```

Comment:
I would prefer not to leave the reader guessing about what exact kind of unsafety there is here (and honestly I prefer the non-double-negative approach also, but nicolas gets to choose that I guess). Can we do maybe `UNSAFE_EMIT_META_EARLY`? Then at least it's clear(er) that the emitting itself isn't unsafe, just doing so earlier than the delayed (safe) case.

Reply:
`UNSAFE_EMIT_META_EARLY` is perfectly fine with me.  (I could probably also be convinced of `EMIT_IRRATIONALLY_OPTIMISTIC_METADATA`.)

Reply:
Changed to add `_EARLY`.